test/Sdk.Generator.Tests/FunctionMetadataProviderGeneratorTests/IntegratedTriggersAndBindingsTests.cs

```diff
@@ -177,6 +189,115 @@ public static IHostBuilder ConfigureGeneratedFunctionMetadataProvider(this IHost
                     expectedOutput);
             }
 
+            [Theory]
+            [InlineData(LanguageVersion.Latest)]
+            public async void FunctionsMultipleOutputBindingWithActionResult(LanguageVersion languageVersion)
+            {
+                string inputCode = """
+                using System;
+                using System.Net;
+                using Microsoft.Azure.Functions.Worker;
+                using Microsoft.Azure.Functions.Worker.Http;
+                using Microsoft.AspNetCore.Http;
+                using Microsoft.AspNetCore.Mvc;
+
+                namespace FunctionApp
+                {
+                    public static class FunctionsMultipleOutputBindingWithActionResult
+                    {
+                        [Function(nameof(FunctionsMultipleOutputBindingWithActionResult))]
+                        public static MyOutputType Run([HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequest req,
+                            FunctionContext context)
+                        {
+                            throw new NotImplementedException();
+                        }
+                    }
+
+                    public class MyOutputType
+                    {
+                        [QueueOutput("functionstesting2", Connection = "AzureWebJobsStorage")]
+                        public string Name { get; set; }
+
+                        [HttpResponseOutput]
```

Comment:
Let's ensure a test is covering the multiple attribute decoration case as well.

Reply:
https://github.com/Azure/azure-functions-dotnet-worker/blob/e0cd2c152acf07b7813af38fac44c20b92704d9b/test/Sdk.Generator.Tests/FunctionMetadataProviderGeneratorTests/DiagnosticResultTests.cs#L106 - these already exist, and refactored so that the properties are checked for multiple bindings before processing potential http attributes.